v1/nodeinfo.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocagent
+
+import (
+	"os"
+
+	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
+)
+
+func createNodeInfo(nodeName string) *commonpb.Node {
+	return &commonpb.Node{
+		Identifier: &commonpb.ProcessIdentifier{
+			HostName:       os.Getenv("HOSTNAME"),
+			Pid:            uint32(os.Getpid()),
+			StartTimestamp: timeToTimestamp(startTime),
+		},
+		LibraryInfo: &commonpb.LibraryInfo{
+			Language: commonpb.LibraryInfo_GO_LANG,
+			Version:  Version,
```

Comment:
This should be the library version. Maybe we need an exporterVersion in the proto definition.

Reply:
Oh, yes, right! I'll make a PR for exporterVersion!

Reply:
Actually we already have `version`
<img width="308" alt="screen shot 2018-09-08 at 2 09 11 pm" src="https://user-images.githubusercontent.com/4898263/45258691-d2413d00-b370-11e8-9828-075b9ca4f0d9.png">

Perhaps we need to properly disambiguate that `version` and call it say `library_version` and then add `exporter_version`.